includes/class-micropub-authorize.php

```diff
@@ -35,8 +38,49 @@ public static function init() {
 		add_action( 'send_headers', array( $cls, 'http_header' ) );
 		add_filter( 'host_meta', array( $cls, 'jrd_links' ) );
 		add_filter( 'webfinger_user_data', array( $cls, 'jrd_links' ) );
+		add_filter( 'rest_index', array( $cls, 'rest_index' ) );
```

Comment:
heh, wow. i read the wordpress docs for `rest_index`, and i still have no idea what it's for. hopefully @pfefferle can help review some of the modern wordpress usage in this PR!

Reply:
Well, rest_index is just an index to allow individuals to know what services are available. Since we are providing indieauth authorization, we are just listing it in the index so people know it is an acceptable method.